src/components/CardUi.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+import { IconButton, CardContent, Chip, CardMedia, Collapse, Card } from '@material-ui/core';
+import {
+  StarRounded as StarRoundedIcon,
+  Room as RoomIcon,
+  ExpandMore as ExpandMoreIcon,
+  StarBorderRounded as StarBorderRoundedIcon
+} from '@material-ui/icons';
+import clsx from 'clsx';
+import Rating from 'react-rating';
+import '../styles/cardUi.scss';
+
+function capFirstLetter(text) {
+  return text.toLowerCase()
+    .split(' ')
+    .map((s) => s.charAt(0).toUpperCase() + s.substring(1))
+    .join(' ');
+}
+
+const CardUi = ({ list }) => {
+
+  const [expanded, setExpanded] = useState(false);
+  const [cardIndex, setCardIndex] = useState(0);
+
+  const handleExpandClick = () => {
+    setExpanded(!expanded);
+  };
+
+  return (
+    <div className="card-container">
+      <Card className="main-card">
+        <div className="card-img">
+          <CardMedia
+            className="card-img-inner"
+            image={process.env.PUBLIC_URL + list[cardIndex].images[0]}
+            title="nandos"
+          />
+        </div>
+        <div className="card-body">
+          <div className="card-title">
+            <h1>
+              {capFirstLetter(list[cardIndex].name)}
+            </h1>
+            <div className="break" />
+            <div className="short-info">
+                <RoomIcon />
+                <span>
+                  {list[cardIndex].location}
+                </span>
+              <Chip label={'$'.repeat(list[cardIndex].price)} className="card-cost" color="secondary" size="medium" variant="outlined" />
```

Comment:
bit long of a line. perhaps break into seperate ones?

Reply:
fixed 22746e9af0c79aaa35981b436f9fb4623e060e38